WalletWasabi.Fluent/Controls/ContentArea.axaml

```diff
@@ -27,18 +27,18 @@
           <ProgressRing Name="LoadingRing" IsIndeterminate="True" IsVisible="{TemplateBinding IsBusy}" />
           <DockPanel Name="MainDockPanel" IsVisible="{Binding !#LoadingRing.IsVisible}">
 
-            <Panel DockPanel.Dock="Bottom" x:Name="PART_ButtonArea">
+            <DockPanel DockPanel.Dock="Bottom" x:Name="PART_ButtonArea" Margin="31 0 31 10">
```

Comment:
The issue with the margin is that you should also change the type in style, so please remove the margin set from the control, and update the style.

https://github.com/WalletWasabi/WalletWasabi/blob/37537da0eb9ed7cccd395a3d8884a2fc1431dee4/WalletWasabi.Fluent/Controls/ContentArea.axaml#L121

Reply:
Fixed